src/commands/insult.ts

```diff
@@ -0,0 +1,69 @@
+import { Client, Message } from 'discord.js';
+import axios from 'axios';
+
+class Users {
+    to: string;
+    from: string;
+    constructor(to: string, from: string){
```

Comment:
Cool typescript feature: If you put 'public' in front of each of the constructor args, you can remove both the two lines above and the two lines below this one.

Basically will do the boilerplate of creating and setting for you if you define that they are public, protected, or private

Reply:
That's pretty neat.  I'm gonna do that now.